jujugui/static/gui/src/app/store/env/api.js

```diff
@@ -2861,6 +2861,9 @@ YUI.add('juju-env-api', function(Y) {
           var response = data.response;
           transformedData.envs = response['user-models'].map(function(value) {
             var model = value.model;
+            if (!model) {
```

Comment:
This shouldn't be possible, can you add a comment as to when this would show up. Also if this is the case I'm not sure returning an array of empty objects is a good idea.